firmware/src/Conduit.cpp

```diff
@@ -12,6 +12,18 @@ device and decide what funciton to all is abstracted away entirely by this libra
 
 #include "Conduit.h"
 
+#define PREFIXED_NAME_CHARS   35
+#define STATIC_STRING_CHARS   64
+#define RECONNECT_DELAY_MS    5000
+#define CONNECT_DELAY_MS      500
+#define ROOT_NODE_NAME        "ROOT"
+#define DEVICE_PATH_SUFFIX    "/device"
```

Comment:
This refactor is not necessarily for easy editing: it is also for clarity when reading through the code. I will add the comment though.